cli/azd/pkg/tools/docker/docker_test.go

```diff
@@ -200,22 +226,28 @@ func Test_DockerBuildArgsMultiple(t *testing.T) {
 	docker := NewDocker(mockContext.CommandRunner)
 
 	mockContext.CommandRunner.When(func(args exec.RunArgs, command string) bool {
-		return strings.Contains(command, "docker build")
+		return strings.Contains(command, "docker build") && !strings.Contains(command, "-q")
 	}).RespondFn(func(args exec.RunArgs) (exec.RunResult, error) {
 		ran = true
 
+		// extract img id file arg
+		argsNoFile, _, value := mocks.RemoveArg("--iidfile", args.Args)
```

Comment:
I'm seeing this in a bunch of places now.  Why is this needed?

Reply:
Can't we just compare using a subset of the array? args[:2]? And also get the `--iidfile` by index as well? These args are all positional right now anyway.